test/associations.test.js

```diff
@@ -0,0 +1,25 @@
+const chai = require("chai");
+const expect = chai.expect;
+const db = require("../server/database/connection.js");
+const models = require("../server/database/models");
+const crypto = require('crypto');
+
+describe('Model Associations', () => {
+
+  before(() => {
```

Comment:
Theres a way in mocha to do the db.sync before all the tests start, may want to consider that, also ensuring that after all tests run, you close the connection to the db.